docs/src/app/home.jsx

```diff
@@ -135,11 +136,13 @@ class Home extends React.PureComponent {
 
                                 <Button
```

Comment:
Can we a Comment here indicating that this is stop gap only until we can truly deprecate all old buttons, at which point this goes away.  Ideally we would like to replace it all .. but obviously that's gonna cause risk.   Would also like to see a ticket for this somewhere

Reply:
Apologies for this question, but what specifically are we talking about here?

Reply:
Maybe add a documentation comment here...?
https://github.com/saddlebackdev/react-cm-ui/pull/325/files#diff-dc9b8e205a6f63db01603b18a6d1f8df41862cd6463bdf63115d839cc9ee2bcbR8